pkgs/development/compilers/llvm/3.9/default.nix

```diff
@@ -54,7 +55,7 @@ let
 
     libcxx = callPackage ./libc++ {};
 
-    libcxxabi = callPackage ./libc++abi.nix {};
+    libcxxabi = callPackage ./libc++abi.nix { cmake = buildPackages.cmake; };
```

Comment:
Do we want to do this for all versions?

Reply:
Actually it wouldn't hurt. I think this is the only place where it's used in cross bootstrapping though.

Reply:
Yeah I think it *should* be done anywhere that callPackage is being overriden like this.

/cc @Ericson2314